src/components/component-playground.less

```diff
@@ -39,6 +39,11 @@
 @checkerboard-square-size: 25px;
 @checkerboard-color: #f1f1f1;
 
+@resizer-border-size: 5px;
+@resizer-width-height: 11px;
+@resizer-border-color: rgba(255, 255, 255, 0);
+@resizer-border-hover-color: rgba(0, 0, 0, 0.5);
```

Comment:
oh man:D you're using less. Let's go with `fade(#000, 50%)` or `darken(#fff, 50%)` :)